src/mesh/mesh_base.C

```diff
@@ -104,6 +104,8 @@ MeshBase::MeshBase (const MeshBase & other_mesh) :
   _skip_find_neighbors(other_mesh._skip_find_neighbors),
   _allow_remote_element_removal(other_mesh._allow_remote_element_removal),
   _elem_dims(other_mesh._elem_dims),
+  _elemset_codes(other_mesh._elemset_codes),
```

Comment:
This is no longer sufficient if we're storing pointers, though, right?  We need a deep copy here?

Reply:
Oh, yeah good catch.